rotkehlchen/chain/evm/tokens.py

```diff
@@ -220,24 +306,66 @@ def detect_tokens(
         - BadFunctionCallOutput if a local node is used and the contract for the
           token has no code. That means the chain is not synced
         """
+        # Check if we should detect tokens for proxies
+        proxies_mapping = {}
+        if self.proxies_module is not None:
```

Comment:
Even though this will go away anyway I have to mention I don't get why you have a `None` check here. You are instantiating this at the constructor.